Userland/Libraries/LibWeb/CSS/Parser/Parser.cpp

```diff
@@ -76,15 +77,232 @@ Vector<QualifiedStyleRule> Parser::parse_as_stylesheet()
         dbgln("");
 
         auto selectors = parse_selectors(rule.m_prelude);
+        CSS::Selector selector = Selector(move(selectors));
+        dump_selector(selector);
     }
 
     return rules;
 }
 
 Vector<CSS::Selector::ComplexSelector> Parser::parse_selectors(Vector<String> parts)
 {
-    (void)parts;
+    // TODO:
+    // This is a mess because the prelude is parsed as a string.
+    // It should really be parsed as its class, but the cpp gods have forsaken me
+    // and i cant make it work due to cyclic includes.
```

Comment:
Cyclic includes?
missing `#pragma once`?

Reply:
Its classes that depends on each other via. a vector
```
// Class B
Vector<ClassA>

// Class A
Vector<ClassB>
```

I tried to forward declare them way back when i first started the parser, but just gave up because it wouldnt work